modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -280,19 +296,24 @@ For [path]``/var/lib/containers/storage/volumes/var-spacewalk``:
 Check the permissions with this command:
 
 ----
-ls -l /var/lib/containers/storage/volumes/var-pgsql /var/lib/containers/storage/volumes/var-spacewalk
+ls -l /var/lib/containers/storage/volumes/var-pgsql
+ls -l /var/lib/containers/storage/volumes/var-spacewalk
 ----
 
 The output should look like this:
 
+// better use?  and change command to display dir, too?
+// drwxr-x--- 1 postgres postgres 48 Apr 19 14:33 _data
+// drwxr-x--- 1 wwwrun   www      30 Apr 19 14:34 _data
+
 ----
 /var/lib/containers/storage/volumes/var-pgsql:
 total 0
-drwxr-x--- 1 10556 10556 48 Apr 19 14:33 _data
+drwxr-x--- 1 postgres postgres 48 Apr 19 14:33 _data
 
 /var/lib/containers/storage/volumes/var-spacewalk:
 total 0
-drwxr-xr-x 1 10552 root 30 Apr 19 14:34 _data
+drwxr-xr-x 1 wwwrun   www      30 Apr 19 14:34 _data
```

Comment:
```suggestion
drwxr-xr-x. 1 10522   root      30 Apr 19 14:34 _data
```

10522 is our hardcoded wwwrun user and from the container host you will see only the number.